terraform/groups/ecs-service/locals.tf

```diff
@@ -13,13 +12,14 @@ locals {
   kms_alias                   = "alias/${var.aws_profile}/environment-services-kms"
   lb_name                     = "alb-randd-rand"
   lb_listener_rule_priority   = 23
-  lb_listener_paths           = ["/ollama-service","/ollama-service/*"]
-  healthcheck_path            = "/ollama-service/api/health" #healthcheck
+  # lb_listener_paths           = ["/ollama-service","/ollama-service/*"]
+  # healthcheck_path            = "/ollama-service/api/health" #healthcheck
+  lb_listener_paths           = ["/","/*"]
```

Comment:
Please correct the service to use a non-root path otherwise will interfere with everything else on the Load balancer - otherwise create a separate Load Balancer for the Ollama service.

Reply:
Or set the lowest priority on the Load Balancer is another option so other rules are prioritised and fall through to this one.